static/src/js/components/OrderStatus/OrderStatusItem.jsx

```diff
@@ -154,14 +154,19 @@ export class OrderStatusItem extends PureComponent {
       retrieval_collection_id: retrievalCollectionId
     } = collection
 
+    const orderStatus = aggregatedOrderStatus(orders)
+
     const [firstOrder = {}] = orders
     const {
-      state,
       type = ''
     } = firstOrder
 
-    // If the order is Harmony and isn't successful, don't show the EDD link
-    if (type.toLowerCase() === 'harmony' && state !== 'successful') return null
+    // If the order is Harmony and is still running or has no files, don't show the EDD link
+    const isDone = !['creating', 'in progress'].includes(orderStatus)
```

Comment:
should these be constants?

Reply:
Theres a const here `sharedUtils/orderStatus.js` but, they aren't specific to Harmony these are apparently the different states that harmony can be in `https://github.com/nasa/harmony/blob/main/db/db.sql#L8`

Reply:
Yes, it should be constants, but I'm not really sure where to start. That orderStatus util @eudoroolivares2016 pointed out has a ton of string values that would be much better served as constants. This file is the same way. 

Rather than working that within this ticket and the scope exploding in size I want to write a new ticket with the goal of cleaning up string values like these.